localstack/localstack-configmap.env

```diff
@@ -1,5 +1,5 @@
 # LOCALSTACK
-LOCALSTACK_URL=0.0.0.0
+LOCALSTACK_URL=http://localstack-service.egs-conv.svc.cluster.local:4566
```

Comment:
are you sure you want not to bind to 0.0.0.0?

Reply:
no, im not sure, it just worked like this